vlib/builtin/README.md

```diff
@@ -0,0 +1,14 @@
+## Description:
+
+`builtin` is a module that...
+
+*TODO*
+
+## Examples:
+
+```v
+import builtin
+
+// TODO: CODE HERE
+
+```
```

Comment:
`builtin` is imported implicitly, and so it is available for every V program

Reply:
it is not like the others, this file is not needed there, or in any case should not have the same cookie cutter structure